src/common/utils.lua

```diff
@@ -62,9 +62,14 @@ end
 ---utils.validateUndername("my-undername")
 ---```
 function utils.validateUndername(name)
```

Comment:
Does this function need any guarding against a nil name?

Reply:
https://github.com/ar-io/ar-io-ant-process/pull/60/commits/186332cb8d8095bc2e271f808b90e244dc35e547